src/dev.sh

```diff
@@ -43,6 +43,9 @@ if [[ "$CURRENT_PLATFORM" == 'windows' ]]; then
     if [[ "$PROCESSOR_ARCHITECTURE" == 'x86' ]]; then
         RUNTIME_ID='win-x86'
     fi
+    if [[ "$PROCESSOR_ARCHITECTURE" == 'ARM64' ]] || [[ "$PROCESSOR_ARCHITECTURE" == 'AMD64' ]]; then
```

Comment:
AMD64 is emulating x64 on an AMD system. I'm happy to remove this if we want to be more explicit about building for arm64, but given git bash on windows 11 arm is currently x64 we would have to manually specify win-arm64 builds when building